lib/Home.dart

```diff
@@ -87,18 +89,45 @@ class _HomeState extends State<Home> {
         ]);
   }
 
+  Widget _noRidesLeftPage (BuildContext context) {
+    return Column (
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Greeting(name: _name),
+        SizedBox(height: 195),
+        Center (
+            child: Column (
+              children: <Widget>[
+                Image(
+                  image: AssetImage('assets/images/steeringWheel@3x.png'),
+                  width: 77,
+                  height: 77,
```

Comment:
Do not hardcode these pixel values. Make them a percentage of the width (or height) by using [MediaQuery.of(context)](https://api.flutter.dev/flutter/widgets/MediaQuery-class.html)

Reply:
Should I just estimate a percentage from the design on Zeplin, like since it's 77 px wide and the screen is 375 px wide on the design, I'll just estimate ~20%?

Reply:
Yes, using ratios is the best way to keep relative sizing consistent!